config/environments/production.rb

```diff
@@ -88,4 +88,14 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.smtp_settings = {
+    domain: 'https://obituary-hub.herokuapp.com/,
```

Comment:
crossing my fingers that this all works smoothly in development.

Reply:
And if not, we can definitely put our heads together and figure it out!